pkg/querier/distributor_queryable.go

```diff
@@ -2,14 +2,16 @@ package querier
 
 import (
 	"context"
+	"sort"
 
+	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/cortexproject/cortex/pkg/prom1/storage/metric"
+	"github.com/cortexproject/cortex/pkg/util/chunkcompat"
```

Comment:
Please leave the blank line so we get three groups of imports.

Reply:
I leave management of imports to my IDE and goimports. Sometimes they get it wrong. Sorry.